### DISCUSSION TITLE
How to change the Cache store when using Cache::tags?

### DISCUSSION THREAD

**Discussion by zengboin** [laravel/framework#47732] [Q&A]
I want to use Cache::tags in different stores like this:
`Cache::store('redis')->tags`,  `Cache::store('array')->tags`  
But from now, I can just use `Cache::tags`, and it use the default store.

What can I do?

**Comment by henzeb** [comment]
I don't understand the question?

**Reply by zengboin** [reply]
From the document: https://laravel.com/docs/10.x/cache#cache-tags
`Cache::tags` only uses the default store,  but I want to use different stores in a request, how can I do it ?

**Reply by zengboin** [reply]
If I use `Cache::store('redis')->tags`, the `Cache::store` return `Illuminate\Contracts\Cache\Repository`, and `Illuminate\Contracts\Cache\Repository` does not define the method `tags`.

**Reply by henzeb** [reply]
Ah, you should use the `driver` method instead. Be aware that not all drivers support tags. `file` doesn't, for example.

**Reply by zengboin** [reply]
I found that: the driver method is the same as store method or just an alias of store.
